docs/install.md

```diff
@@ -22,9 +22,20 @@ make helm-upgrade-keda
 
 >This document will rely on environment variables such as `${NAMESPACE}` to indicate a value you should customize and provide to the relevant command. In the above `helm install` command, `${NAMESPACE}` should be the namespace you'd like to install KEDA into. KEDA must be installed in every namespace that you'd like to host KEDA-HTTP powered apps.
 
+## Install via Helm Chart
+
+This repository is within KEDA's default helm repository on [kedacore/charts](http://github.com/kedacore/charts), you can install it by running:
+
+```console
+helm repo add kedacore https://kedacore.github.io/charts
+helm repo update
+
+helm install http-add-on kedacore/http-add-on --create-namespace --namespace keda
```

Comment:
Let's update with the new name, once merged

Reply:
I changed this name on the charts as well and in the latest commit